packages/components/psammead-grid/README.md

```diff
@@ -0,0 +1,176 @@
+
+## Description
+
+
+## Installation
+
+`npm install @bbc/psammead-grid`
+
+## Props
+
+<!-- prettier-ignore -->
+| Argument  | Type | Required | Default | Example |
+| --------- | ---- | -------- | ------- | ------- |
+| columns | object | yes | N/A | { group1: 6, group3: 8 } |
+| startOffset | object | no | Sets all values as 1 for each of the groups defined in `columns`  | { group1: 2, group3: 3 } |
+| wrapper | boolean | no | false | `wrapper` |
+| enableGelGutters | boolean | no | false | `enableGelGutter` |
+| enableGelMargins | boolean | no | false | `enableGelMargins` |
+| enableGelMaxWidths | boolean | no | false | `enableGelMaxWidths` |
+| enableGroupFourMaxWidth | boolean | no | false | `enableGroupFourMaxWidth` |
+
+## Usage
+
+We use the GEL Grid for our breakpoints, as well as for our standard gutters and margins. [The values are defined here in the GEL Grid documentation.](https://www.bbc.co.uk/gel/guidelines/grid#grid-sizes).
+
+When we refer to `group3` in this component, we're referring to the breakpoint with a minimum width `GEL_GROUP_3_SCREEN_WIDTH_MIN` & a maximum width `GEL_GROUP_3_SCREEN_WIDTH_MAX`. These values are defined within [`@bbc/gel-foundations/breakpoints`](https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/src/breakpoints.js).
+
+####<a name="simple-example">Simple example</a>
+
+Here is an example of a `Grid` `wrapper` that has 8 columns for `group4` (from 1008px to 1279px). It has two child `Grid` elements, one which spans 6/8 columns and one which spans 2/8 columns within this breakpoint.
+
+```jsx
+import Grid from '@bbc/psammead-grid';
+
+const MyComponent = () => (
+  <Grid wrapper columns={{ group4: 8 }}>
+    <Grid columns={{ group4: 6 }}>
+      <p>Paragraph that spans 6 out of 8 columns through group4</p>
+    </Grid>
+    <Grid columns={{ group4: 2 }}>
+      <p>Paragraph that spans 2 out of 8 columns through group4</p>
+    </Grid>
+    <Grid columns={{ group4: 2 }}>
+       <p>Paragraph that spans 2 out of 8 columns through group4</p>
+    </Grid>
+    <Grid columns={{ group4: 2 }}>
+       <p>Paragraph that spans 2 out of 8 columns through group4</p>
+    </Grid>
+  </Grid>
+);
+```
+
+####<a name="multiple-groups">Setting values for multiple groups</a>
+
+```jsx
+import Grid from '@bbc/psammead-grid';
+
+const MyComponent = () => (
+  <Grid wrapper columns={{ group3: 6, group4: 8 }}>
+    <Grid columns={{ group3: 6, group4: 6 }}>
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 6/8 columns</p>
+    </Grid>
+    <Grid columns={{ group3: 6, group4: 2 }}>
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 2/8 columns</p>
+    </Grid>
+  </Grid>
+);
+```
+
+[Screenshot of this example without GEL Gutters and GEL Margins](./documentation/Screenshot\ example\ without\ gelMargins\ and\ gelGutters.png)
```

Comment:
Needs fixed

Reply:
The image has had its syntax updated, but since I've used a relative path, it's pointing to the file location on the `latest` branch, so isn't visible on the PR just yet. Haven't yet found a work-around for this.